### DISCUSSION TITLE
什麼時候 cue MBTI

### DISCUSSION THREAD

**Discussion by forestdew507** [1101-GS4719-C/Python-Community-GS4719-C#2278] [General]
剛閱讀完一篇 github 的 MBTI 相關討論，於是寫下這篇...

我曾經在面試的時候遇到有高中生自介開頭幾句就說，我的 MBTI 是 XXXX，當下我真的差點笑出來。不得不說，在 MBTI 剛紅起來的時候我還不覺得它具有什麼效力，但隨著後來跟不同的人討論，才覺得好像還是有一些參考價值XD

首先，聽說在傳播學院 ( 恩對中央沒有所以可能比較感受不到 ? 還是其實也有其他院的 MBTI 算是蠻普及的 ? )，各種招工報名表單常有 MBTI 一欄。
再者，關於面試時提及 MBTI。即使世界上的人怎麼可能能夠以 16 種去分 ( 這句話應該聽過很多次了哈哈 包含星座 12 種人這種說法 ) 但似乎不得不承認，可以用這 16 種的大類去「快速掌握」一個人的大方向，也許 ( 我是說也許 ) 將有助於加速接下來的社交討論等。

**Comment by christaaah** [comment]
我覺得這種性格測驗大方向是蠻可信的，因為在做測驗的時候就已經能看出來面對類似情況下自己會做出什麼舉動的傾向了，但是我覺得凡事都扯上MBTI或者一句話就概括MBTI的那些言論還滿好笑的，但其實某些方面又挺有趣的，因為很多時候和朋友之間談論這些並不是因為想要精準測出自己的性格，而是當作一種聊天的話題或是調侃的方法，就像哪種網站上的心理測驗一樣。

**Comment by Mminniehuang** [comment]
最近跟朋友約出去玩的時候就有一段時間大家在猜對方的MBTI，我覺得蠻有趣的，過程中大家就對某個人的認識開始分析和討論猜測，猜對的時候就會一片贊同，錯的時候就會開始大笑調侃本人或是說測這個不准啦某某怎麼可能是這個這個。說不定MBTI除了是快速認識一個人的方法，也可以是更認識某位朋友的有趣遊戲。

**Comment by Elaine4488** [comment]
我覺得你提到的“快速掌握一個人的大方向”還挺有道理的，畢竟MBTI確實能夠提供一些關於人際互動、工作風格等方面的線索。但的確，當人們了解了一些基本的MBTI分類後，會習慣性地用這些標籤來試圖理解自己或他人。
